torch/jit/frontend.py

```diff
@@ -156,6 +156,10 @@ def get_class_properties(cls, self_name):
 
     return properties
 
+def get_unresolved_class_attributes(cls, ast):
```

Comment:
Currently this function returns something like `['__module__', 'attr', '__dict__', '__weakref__', '__doc__']` 
I think you mean we should get rid of builtin methods with __ prefix right?

To ensure all unresolved class attrs are what user specified in the class, I agree it's better to parse the class on our own instead of relying on python semantics.

Do we have a standalone parser for doing this? I'm not sure whether `inspect` module will return an ast.